source/SubsystemTests/SubsystemTestFixture.cs

```diff
@@ -158,6 +159,15 @@ public SubsystemTestFixture()
             "MARKET_PARTICIPANT_URI",
             defaultValue: "https://app-webapi-markpart-u-001.azurewebsites.net"));
 
+        EdiFunctionAppUri = new Uri(
+            GetConfigurationValue<string>(
+                root,
+                "func-edi-api-base-url"));
+
+        EdiApplicationId = GetConfigurationValue<string>(
```

Comment:
```suggestion
        EdiApplicationIdUri = GetConfigurationValue<string>(
```

We've decided to call it "application id uri" (since that is what it is called in Azure), so let is keep that naming to avoid confusion.